CHANGELOG.md

```diff
@@ -1,3 +1,31 @@
+# 0.33.0 (2020-10-12)
+
+[CKB v0.35.1](https://github.com/nervosnetwork/ckb/releases/tag/v0.35.1) was released on Sept. 14th, 2020. This version of CKB node is now bundled and preconfigured in Neuron. 
+
+**Note:** There will be a data migration for the new bundled node [CKB v0.35.1](https://github.com/nervosnetwork/ckb/releases/tag/v0.35.1), which may take several minutes to boot up for the first time after upgraded from the earlier versions. 
+**Attention:** The migration is not reversible. Please backup the data directory if you want to use old versions later.
+
+### Refactor
+
+* Upgrade from Electron 7 to Electron 9.
+* Refactor the wallet address storage to be based on an SQLite table rather than a JSON file.
+* Deprecate LevelDB and use SQLite instead.
```

Comment:
_Deprecate LevelDB and use SQLite instead._ => _Deprecate LevelDB and instead  SQLite for persisting transaction descriptions._

Reply:
updated.